mqtt/mqtt-broker/tests/common.rs

```diff
@@ -44,37 +45,90 @@ impl TestClient {
         self.publish_handle.publish(publication).await
     }
 
+    pub async fn publish_qos0(&mut self, topic: &str, payload: &str, retain: bool) {
```

Comment:
```suggestion
    pub async fn publish_qos0(&mut self, topic: impl Into<String>, payload: impl Into<Bytes>, retain: bool) {
```
because https://docs.rs/bytes/0.5.4/bytes/struct.Bytes.html#impl-From%3C%26%27static%20%5Bu8%5D%3E